packages/common/lib/types/StateManager.types.ts

```diff
@@ -4,6 +4,11 @@ export interface CredentialOfferState {
   clientId?: string;
   credentialOffer: CredentialOfferPayloadV1_0_11;
   createdOn: number;
+  'pre-authorized_code': string;
+  preAuthorizedCodeCreatedOn: number;
```

Comment:
Why is this property needed if there already is a createdOn?

Reply:
Fixed!